vl-proza-message.src.js

```diff
@@ -136,6 +154,76 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
             this._element.appendChild(this._getEditButtonTemplate());
         }
     }
+
+    __initWysiwyg() {
+        this.__unwrapWysiwygElement();
+        tinyMCE.baseURL = '/node_modules/tinymce';
+        tinyMCE.init({
+            target: this._wysiwygElement,
+            menubar: false,
+            inline: true,
+            toolbar: false,
+            plugins: ['quickbars'],
+            quickbars_selection_toolbar: 'bold italic underline',
+            powerpaste_word_import: 'clean',
+            powerpaste_html_import: 'clean',
+            content_css: '/style.css',
+            verify_html: false,
+            suffix: '.min'
+        });
+        tinyMCE.activeEditor.on('init', () => {
+            const editor = tinyMCE.activeEditor;
+            editor.focus();
+            editor.selection.select(editor.getBody(), true);
+            editor.selection.collapse(false);
+            editor.bodyElement.classList.add('vl-typography');
+            this._buttonElement.remove();
+            editor.on('keydown', (e) => {
+                if (e.keyCode === 27) {
+                    while (editor.undoManager.hasUndo()) {
+                        editor.undoManager.undo();
+                    }
+                    this.__stopWysiwyg(editor);
+                }
+                if (e.keyCode === 13 && !e.shiftKey) {
+                    this.__stopWysiwyg(editor);
+                }
+            });
+            editor.on('blur', () => {
+                this.__stopWysiwyg(editor);
+            });
+        });
+    }
+
+    __stopWysiwyg(editor) {
+        editor.destroy();
+        this._element.appendChild(this._getEditButtonTemplate());
```

Comment:
een reden waarom ge de button weg doet en terugzet ipv te hiden?

Reply:
In ieder geval niet via class en CSS in kader van WCAG. Ik zal de implementatie vervangen door gebruik te maken van het hidden attribuut.